### DISCUSSION TITLE
Azure sql server

### DISCUSSION THREAD

**Discussion by CelsoMMiranda** [getredash/redash#6489] [Q&A]
Hello, I have a database instantiated within Azure SQL Server, but I cannot connect. Has anyone gone through something similar?

**Comment by doprado** [comment]
Hello, I also have the same problem. I already performed for v10.1 and was unsuccessful. When I try to connect via ODBC, Nginx crashes.

My database is: Microsoft SQL Azure (RTM) - 12.0.2000.8 Oct 10 2023 09:37:59

I've tried everything, before I ran it without problems on AWS RDS. After migrating to Azure I can no longer do so.

All my other apps work, only Redash doesn't.
